src/item.h

```diff
@@ -495,6 +495,13 @@ class Item : virtual public Thing
 			return getIntAttr(ITEM_ATTRIBUTE_CORPSEOWNER);
 		}
 
+		void setRewardCorpse() {
+			setCorpseOwner(static_cast<uint32_t>(std::numeric_limits<int32_t>::max()));
```

Comment:
What it is supposed to mean is in the member name. Don't you think this looks redundant ?

````C++
void setRewardCorpse() {
    setCorpseOwner(REWARD_CORPSE_OWNER);
}
````